### PR TITLE
No Bug: Run bootstrap before all TestFlight builds by default

### PR DESCRIPTION
**Pull request by kylehickinson**
## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [ ] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [ ] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -16,6 +16,11 @@ platform :ios do
       UI.message "Running git status check"
       ensure_git_status_clean(show_uncommitted_changes: true, show_diff: true)
     end
+    if lane != :test
+      unless options[:skip_bootstrap]
+        sh("pushd .. && ./bootstrap.sh && popd")
```
Comment (kylehickinson) [inline fastlane/Fastfile]: Actually let me change this to include the `--ci` option

File: fastlane/Fastfile [hunk 2]
```diff
@@ -16,6 +16,11 @@ platform :ios do
       UI.message "Running git status check"
       ensure_git_status_clean(show_uncommitted_changes: true, show_diff: true)
     end
+    if lane != :test
+      unless options[:skip_bootstrap]
```
Comment (iccub) [inline fastlane/Fastfile]: Can be written simpler `unless lane == :test && options[:skip_bootstrap]`

File: fastlane/Fastfile [hunk 3]
```diff
@@ -16,6 +16,11 @@ platform :ios do
       UI.message "Running git status check"
       ensure_git_status_clean(show_uncommitted_changes: true, show_diff: true)
     end
+    if lane != :test
```
Comment (iccub) [inline fastlane/Fastfile]: Don't we want to run it before git status check?
To make sure bootstrap does not make our repo dirty

**Review by iccub — CHANGES_REQUESTED** [review]

File: fastlane/Fastfile (hunk 3 above)
Comment (kylehickinson) [reply fastlane/Fastfile]: Good idea!

**Review by iccub — APPROVED** [review]

**Review by iccub — APPROVED** [review]

**Pull request merged**